src/base/random.test.cc

```diff
@@ -84,7 +84,7 @@ TEST(RandomCtor, DoubleDefaultConstruct)
     // First double corresponds to the default seed
     // 14514284786278117030llu / std::numeric_limits<uint64_t>::max()
     ASSERT_EQ(dut->random<double>(),
-      0.7868209548678020137657540544751100242137908935546875d);
+      0.7868209548678020137657540544751100242137908935546875);
```

Comment:
May still need the trailing "d" so that the compiler doesn't truncate.

Reply:
In CPP literal floating-points are double. The "F" is used to force float and "L" is used to force a long double.

"d" isn't an official CPP convention though GCC permits it. Clang does not which is how I found this bug.